ducktape/command_line/main.py

```diff
@@ -35,9 +39,23 @@ def parse_args():
                         help='one or more space-delimited strings indicating where to search for tests')
     parser.add_argument("--collect-only", action="store_true", help="display collected tests, but do not run")
     parser.add_argument("--debug", action="store_true", help="pipe more verbose test output to stdout")
+    parser.add_argument("--cluster", action="store", default="ducktape.cluster.vagrant.VagrantCluster",
```

Comment:
This could be a future issue, but it could be useful to specify config file with a flag.